src/SourceLocator/Type/AbstractSourceLocator.php

```diff
@@ -27,7 +28,7 @@
      * @param Identifier $identifier
      * @return LocatedSource
```

Comment:
The docblock says not nullable - is this a BC break or a broken previous assumption?